kv/txn_coord_sender.go

```diff
@@ -758,28 +756,26 @@ func (tc *TxnCoordSender) heartbeat(id string) {
 					Reply: reply,
 				}
 
-				task := tc.stopper.StartTask()
-				if !task.Ok() {
+				if !tc.stopper.RunTask(func() {
```

Comment:
@tamird What would you propose instead? These return values are serving a purpose and I don't think it will be trivial to remove them. I definitely do not think that we should try to solve that problem in this PR; removing the return values will require scrutiny of each place where they were used so it should get its own PR.